tests/modules/individuals/test_merge.py

```diff
@@ -17,23 +17,15 @@
     reason='Individuals module disabled',
 )
 def test_merge(db, flask_app_client, researcher_1, request, test_root):
+    import uuid
+
     from app.modules.individuals.models import Individual
 
     sighting_data = {
-        'encounters': [
-            {
-                'locationId': 'one',
-            },
-            {
-                'locationId': 'two',
-            },
-            {
-                'locationId': 'three',
-            },
-        ],
+        'encounters': [{}, {}, {}],
```

Comment:
@hwindsor per your original question about this (which seems **gone** to me now -- what is up with that???), it turns out we totally _can_ just have empty-defined encounters like this.  i made the change and the test passes.  so i am going with this instead.